### PR TITLE
feat(EditServiceModal): only show framework edit if hook true

### PR DESCRIPTION
**Pull request by ahoskins**
This hook is added because the API is EE for now. The `cosmos/service/update` API is not yet migrated.

Closes DCOS-19784.

**Checklist**
- [ ] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

**Comment by ahoskins** [comment]
Hold merge till I fix these tests.

**Comment by MatApple** [comment]
@ahoskins let's disable the tests so that it's easy to reinstate them again when we remove this hook.

File: tests/pages/services/ServiceActions-cy.js [hunk 1]
```diff
@@ -57,153 +57,7 @@ describe("Service Actions", function() {
...
-    it("shows edit config button on review screen that opens form", function() {
-      cy.get(".modal .modal-header button").contains("Review & Run").click();
-
-      cy.get(".modal button").contains("Edit Config").click();
-
-      cy
-        .get('.modal .menu-tabbed-container input[name="name"]')
-        .should("to.have.value", "elastic");
-    });
+    // re-add test from https://github.com/dcos/dcos-ui/commit/087cbe0a191e48685d82d087a5723088bef4777b#diff-aaa1f5e99d424cc576e3a866c4d89c3e
```
Comment (orlandohohmeier) [inline tests/pages/services/ServiceActions-cy.js]: ➡️  Please file an issue and the number here.

**Comment by ahoskins** [comment]
@orlandohohmeier I've added a link to a Jira ticket for us to renable this.

**Pull request merged**